ui/app/components/http-requests-bar-chart.js

```diff
@@ -138,9 +153,34 @@ export default Component.extend({
       .merge(barsEnter)
       .attr('width', xScale.bandwidth())
       .attr('height', counter => height - yScale(counter.total))
-      // the offset between each bar
       .attr('x', counter => xScale(counter.start_time))
       .attr('y', counter => yScale(counter.total));
+
+    bars.exit().remove();
+
+    // render transparent bars and bind the tooltip to them
+    // since we cannot bind the tooltip to the actual bars
```

Comment:
If binding to the actual bars is not possible, sounds like a nice workaround! Nice decision to put a comment in here also!

Reply:
+1 on the comment - maybe call out clip path for being the reason (what a weird thing though 😬 )